stdlib/2and3/xml/sax/saxutils.pyi

```diff
@@ -10,9 +10,9 @@ def quoteattr(data: Text, entities: Mapping[Text, Text] = ...) -> Text: ...
 
 class XMLGenerator(handler.ContentHandler):
     if sys.version_info >= (3, 0):
-        def __init__(self, out=..., encoding=..., short_empty_elements: bool = ...) -> None: ...
+        def __init__(self, out: TextIO = ..., encoding: str = ..., short_empty_elements: bool = ...) -> None: ...
```

Comment:
`out` may also be None or a wide variety of other file-like objects (see the implementation of _gettextwriter).